File: python/questao3.py
def remove_spaces(input_string):
    return ''.join(input_string.split())

def main():
    input_string = input("Digite uma string: ")

    result_string = remove_spaces(input_string)

    print("String resultante após remover os espaços:", result_string)

if __name__ == "__main__":
    main()

File: c/questao3.c
#include <stdio.h>
#include <string.h>

void removeSpaces(char* str) {
    int n = strlen(str);
    int count = 0;

    for (int i = 0; i < n; i++) {
        if (str[i] != ' ') {
            str[count++] = str[i];
        }
    }
    str[count] = '\0';
}

int main() {
    char str[100];

    printf("Digite uma string: ");
    fgets(str, sizeof(str), stdin);

    removeSpaces(str);

    printf("String resultante após remover os espaços: %s\n", str);

    return 0;
}
